src/Platforms/AbstractMySQLPlatform.php

```diff
@@ -452,12 +452,16 @@ private function buildTableOptions(array $options): string
 
         $tableOptions[] = sprintf('DEFAULT CHARACTER SET %s', $options['charset']);
 
-        // Collate
-        if (! isset($options['collate'])) {
-            $options['collate'] = $options['charset'] . '_unicode_ci';
+        if (isset($options['collate'])) {
```

Comment:
If the collate and collation keys are both defined, here collate takes the precedence over collation.
If we want to deprecate collate in favor of collation I'd do the opposite.

Reply:
Maybe I should instead throw if both are defined? Or if both are defined but different? I don't think that it would make sense to have both defined but different.

Reply:
I like the idea of throwing if they are different. It would also be nice to trigger a deprecation warning if `collate` is defined.

Reply:
I'd refrain from throwing new exceptions and triggering deprecations in a patch release. It's also fine that "collate" takes precedence thereby providing less disruption during the upgrade.